src/codegen/codegen_info.hpp

```diff
@@ -253,6 +253,12 @@ struct CodegenInfo {
     /// typically equal to number of primes
     int num_equations = 0;
 
+    /// number of ppvar variables (ints and pointers)
+    /// In \c nocmodl it's calculated like the following:
+    /// ppvar_count = ioncount + diamdec + pointercount + areadec + net_send_seen_
+    /// + watch_seen_ + for_netcons_ + cvode_emit
+    int ppvar_count;
```

Comment:
as mentioned above, I am curious if this can be removed. Otherwise, at least I would rename `ppvar` to something that conveys some meaning for new people?

Reply:
Done